Add progress indicators

snrubnomis opened the pull request:
Add table to show progress of game (motive, means, opportunity)

Make UI full height

simon-burns committed:
Add progress indicators

Add table to show progress of game (motive, means, opportunity)

Make UI full height

jt-nti commented:
Expanding the progress table hides the messages at the bottom of the chat window.

simon-burns committed:
Scroll chat to bottom when opening progress

Ensure chat stays scrolled to the bottom when opening and closing
progress section
